src/app/components/shared/BreadCrumbs/BreadCrumbs.tsx

```diff
@@ -1,80 +1,38 @@
-
 import styled from "styled-components"
 import { Link, useParams } from "@reach/router"
-import { Icon, themeSpacing, themeColor, ascDefaultTheme } from "@amsterdam/asc-ui"
-import { ChevronRight } from "app/components/shared/Icons"
+import { Breadcrumbs } from "@amsterdam/asc-ui"
 import to from "app/routing/utils/to"
 import find from "app/routing/utils/find"
 import routes from "app/routing/routes"
 
-type Props = {
-  routeParams?: Record<string, unknown>
+type BreadCrumbsItem = {
+  title: string
+  to: string
+  path: string
 }
 
-const LEVEL = "level5"
-const Ul = styled.ul`
-  margin: 0;
-  list-style: none;
-  padding-left: 0;
-  line-height: 1.6em;
-  li {
-    display: inline;
-    font-size: 14px;
-    a, span {
-      color: ${ themeColor("tint", LEVEL) };
-    }
-    a {
-      text-decoration: none;
-      &:hover {
-        color: ${ themeColor("secondary") };
-        text-decoration: underline;
-      }
-    }
-  }
-  
-`
-const StyledSeperator = styled(Icon)`
-  display: inline;
-  margin: 0 ${ themeSpacing(1) };
-  svg {
-    vertical-align: text-bottom;
-  }
+const StyledBreadcrumbs = styled(Breadcrumbs)`
+  padding-inline-start: 0;
```

Comment:
Why not use `padding-left`

Reply:
It's to override the default browser value